addons/test_mail/models/test_mail_models.py

```diff
@@ -112,3 +112,25 @@ class MailModel(models.Model):
     def _value_pc(self):
         for record in self:
             record.value_pc = float(record.value) / 100
+
+
+class MailTestPartnerOnUpdate(models.Model):
```

Comment:
Maybe we could have 1 class and play with patcher to redefine the output of various methods to override. That way we simplify test models.